src/signal.rs

```diff
@@ -129,32 +99,27 @@ fn SIGRTMAX() -> c_int {
 
 /// Verify that a signal number is valid.
 ///
-/// VCPU signals must be enclosed within the OS
-/// limits for realtime signals. The remaining ones need
+/// Usually, VCPU signals are assigned and must be enclosed within the OS
+/// limits for realtime signals, while the remaining ones need
 /// to be between the minimum (`SIGHUP`) and maximum (`SIGSYS`) values.
```

Comment:
**must be enclosed** is incorrect.

Let's make the whole comment clearer:
```rust
/// Supported signals range from `SIGHUP` to `SIGUSR2` and from `SIGRTMIN` to `SIGRTMAX`.
/// We recommend using realtime signals `[SIGRTMIN, SIGRTMAX]` for VCPU threads.
```

Reply:
👍 done that.